includes/polling/sensors/runtime/apc.inc.php

```diff
@@ -0,0 +1,13 @@
+<?php
+/*
+ * LibreNMS
+ *
+ * Copyright (c) 2016 Cercel Valentin <crc@nuamchefazi.ro>
+ * This program is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the
+ * Free Software Foundation, either version 3 of the License, or (at your
+ * option) any later version.  Please see LICENSE.txt at the top level of
+ * the source code distribution for details.
+ */
+
+$sensor_value = snmp_get($device, $sensor['sensor_oid'], '-OUqnvt');
```

Comment:
that "t" flag does the trick, without it the sensor return time in format: 0:0:13:00, with that one we got time-ticks, which are used on discovery process (and also all the divisors/multipliers are based on time-ticks)